data/governance.yml

```diff
@@ -0,0 +1,454 @@
+      kind: github
+    members:
+    - name: Marek Kubica
+      github: Leonidas-from-XIV
+      role: Lead Maintainer
+  - id: ocp-indent-dev
+    name: Ocp-indent
+    description: The Ocp-indent development team
+    contacts:
+    - name: GitHub
+      link: https://github.com/OCamlPro/ocp-indent
+      kind: github
+    members:
+    - name: Louis Gesbert 
+      github: AltGr
+      role: Lead Maintainer
+  - id: ocamlfind-dev
+    name: Ocamlfind
+    description: The Ocamlfind development team
+    contacts:
+    - name: GitHub
+      link: https://github.com/ocaml/ocamlfind
+      kind: github
+    members:
+    - name: Gerd Stolpmann
+      github: gerdstolpmann
+      role: Lead Maintainer
+    - name: David Allsopp
+      github: dra27
+      role: Maintainer
+  - id: omp-dev
+    name: Omp
+    description: The Omp development team
+    contacts:
+    - name: GitHub
+      link: https://github.com/ocaml-ppx/ocaml-migrate-parsetree
+      kind: github
+    members:
+    - name: Sonja Heinze
+      github: pitag-ha
+      role: Lead Maintainer
+- id: packaging
+  name: Packaging
+  description: The Packaging team is responsible to ensure the quality of the OCaml
+    packages ecosystem, and in particular, to review submitions to the Opam repository.
+  contacts:
+  - name: GitHub
+    link: https://github.com/ocaml/opam-repository
+    kind: github
+  members:
+  - name: Anil Madhavapeddy
+    github: avsm
+    role: Lead Maintainer
+  - name: Kate Deplaix
+    github: kit-ty-kate
+    role: Maintainer
+  - name: Marcello Seri
+    github: mseri
+    role: Maintainer
+- id: infrastructure
+  name: Infrastructure
+  description: The infrastructure team is responsible to maintain and evolve the infrastructure
+    powering the official OCaml projects, including the servers for OCaml.org and
+    the opam-repository CI.
+  contacts:
+  - name: Mailing List
+    link: https://lists.ocaml.org/listinfo/listinfo/infrastructure
+    kind: email
+  - name: GitHub
+    link: https://github.com/ocaml/infrastructure/issues
+    kind: github
+  members:
+  - name: Anil Madhavapeddy
+    github: avsm
+    role: Owner
+  - name: Tim McGilchrist
+    github: tmcgilchrist
+    role: Lead Maintainer
+  - name: Kate
+    github: kit-ty-kate
+    role: Maintainer
+  - name: Mark Elvers
+    github: mtelvers
+    role: Maintainer
+- id: web
+  name: Web
+  description: The Web team is responsible for the development and maintenance of
+    the official OCaml websites, in particular OCaml.org and its subdomains.
+  contacts:
+  - name: GitHub
+    link: https://github.com/ocaml/ocaml.org
+    kind: github
+  members:
+  - name: Anil Madhavapeddy
+    github: avsm
+    role: Owner
+  - name: Thibaut Mattio
+    github: tmattio
+    role: Lead Maintainer
+  - name: Sabine Schmaltz
+    github: sabine
+    role: Maintainer
+  - name: Cuihtlauac Alvarado
+    github: cuihtlauac
+    role: Maintainer
+working-groups:
```

Comment:
@aantron do you want to add a working group for Dream here?
@anmonteiro how about one for Melange?

Reply:
I'd be OK adding a working group for Melange.

Perhaps including myself and (at least) @jchavarri?

Reply:
Is there a dev meeting or chat for the Melange working group? @anmonteiro

Reply:
There is the [Reason discord](https://discord.gg/reasonml), where there is a specific `#melange` channel.